internal/logging/logging.go

```diff
@@ -27,3 +30,20 @@ func configureZeroLogger() {
 		return file + ":" + strconv.Itoa(line)
 	}
 }
+
```

Comment:
```suggestion
NewLogWriter returns a writer for the given logDir and logFile
If the log file already exists it will be removed and a fresh file will be created
```